DuckDuckGo.xcodeproj/xcshareddata/xcschemes/DuckDuckGo.xcscheme

```diff
@@ -108,7 +108,7 @@
          <EnvironmentVariable
             key = "VARIANT"
             value = "mb"
-            isEnabled = "YES">
+            isEnabled = "NO">
```

Comment:
Just removing Core from the list of selectable Schemes since it can't do anything